Task Page: Data Brokering

Nazeeefa opened the pull request:
**Data Brokering**
New task page, by [ELIXIR Converge WP9 team](https://elixir-europe.org/about-us/how-funded/eu-projects/converge/wp9), that addresses how to get started as a data broker; how data and metadata collection and sharing should be done. The page covers key considerations, and provides solutions and recommendations that could help with how to share data within a well defined ethical and legal framework.

**Are there existing pages in the RDMkit website related to the requested page?**
The 'Data Brokering' task page can be added under "related pages" in the following page: [Your Role - Data Steward: infrastructure](https://rdmkit.elixir-europe.org/data_steward_infrastructure)

bedroesb commented:
Hey Nazeefa! I changed the image filename so it does not contain (1) + updated the image syntax to be markdown.

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+---
+title: Data brokering
+search_exclude: true
```
```suggestion
```

when this gets approved the exclude can go away :)

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Identify what kind of processing you will handle as a broker, such as (meta)data curation and validation, data masking/anonymisation, etc.
+* Define the time frame for your commitment and your responsibilities for the data, such as how to handle data loss before delivery, what to do with the data after a successful delivery, how to manage changes to data that has already been delivered etc.
+* Identify who is responsible for the data before, during and after delivery, such as the data controller/processor (according to GDPR) and/or intellectual property owner/licensee relationships between the provider and recipient
+* Ensure that you will be able to establish contracts/agreements that cover the data and processing that you will handle, such as considerations for [data protection](https://rdmkit.elixir-europe.org/data_protection.html), [licensing](https://rdmkit.elixir-europe.org/licensing.html) and [compliance](https://rdmkit.elixir-europe.org/compliance_monitoring.html).
```
```suggestion
* Ensure that you will be able to establish contracts/agreements that cover the data and processing that you will handle, such as considerations for [data protection](data_protection), [licensing](licensing) and [compliance](compliance_monitoring).
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+### Solutions
+
+The solutions that you adopt will vary depending on the agreements you have negotiated with data providers and/or recipients. The following are examples of general solutions that would help you comply with regulations and implement good data management practices.
+* [Data management plan](https://rdmkit.elixir-europe.org/data_management_plan.html) – Many questions that you would answer while writing a data management plan can be relevant to answer when you specify the terms of service for your brokering service, such as data storage, data standards, legal and ethical, etc. 
```
```suggestion
* [Data management plan](data_management_plan) – Many questions that you would answer while writing a data management plan can be relevant to answer when you specify the terms of service for your brokering service, such as data storage, data standards, legal and ethical, etc. 
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+
+The solutions that you adopt will vary depending on the agreements you have negotiated with data providers and/or recipients. The following are examples of general solutions that would help you comply with regulations and implement good data management practices.
+* [Data management plan](https://rdmkit.elixir-europe.org/data_management_plan.html) – Many questions that you would answer while writing a data management plan can be relevant to answer when you specify the terms of service for your brokering service, such as data storage, data standards, legal and ethical, etc. 
+* [Data protection](https://rdmkit.elixir-europe.org/data_protection.html) – If you are working with data concerning people in the EU, you should make sure to comply with both national and international regulations for data protection.
```
```suggestion
* [Data protection](data_protection) – If you are working with data concerning people in the EU, you should make sure to comply with both national and international regulations for data protection.
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+Data brokering involves collecting data from various data providers (metadata and other data e.g. sequencing files), standardising and curating the data if needed, and then preparing the data for re-sharing e.g. in public international repositories. On the brokering platform, it is recommended that data be stored in a structured manner (e.g. in a relational database), using as much as possible co ...
+
+### Considerations
+* [Data collection](https://rdmkit.elixir-europe.org/collecting) should be carefully prepared, notably to define the data model, the metadata and data that are needed for the envisioned applications, assess which fields should be compulsory or optional, follow controlled vocabularies or ontologies, and identify the nature of data (personal, sensitive data) and thereby the required level of securi ...
```
```suggestion
* [Data collection](collecting) should be carefully prepared, notably to define the data model, the metadata and data that are needed for the envisioned applications, assess which fields should be compulsory or optional, follow controlled vocabularies or ontologies, and identify the nature of data (personal, sensitive data) and thereby the required level of security or data treatment (e.g. pseudonymised or anonymised data, ethical consent…).
```

Nazeeefa replied:
are 'and' and ''or' highlighted in red means they will be excluded in final changes?

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+
+### Considerations
+* [Data collection](https://rdmkit.elixir-europe.org/collecting) should be carefully prepared, notably to define the data model, the metadata and data that are needed for the envisioned applications, assess which fields should be compulsory or optional, follow controlled vocabularies or ontologies, and identify the nature of data (personal, sensitive data) and thereby the required level of securi ...
+  * Collection of *metadata* can be done in various ways, each having its advantages and disadvantages, notably in terms of user-friendliness, ease of processing and [data quality](https://rdmkit.elixir-europe.org/data_quality.html).
```
```suggestion
  * Collection of *metadata* can be done in various ways, each having its advantages and disadvantages, notably in terms of user-friendliness, ease of processing and [data quality](data_quality).
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+
+* Regular data checks should always be performed at the database level, to check unicity of identifiers where expected (e.g. sample identifier within a laboratory is expected to be unique) and identify potential incoherences in the data (e.g. division indicated in the name of a virus versus division indicated in the location field).
+* Data brokers should also consider implementing a mechanism for metadata and data files update within their platform, and define mechanisms to pass on the updates to international repositories. Please consider that depending on the update mechanisms in place at the international repositories (e.g. API vs manual update via email), this process might become quite time-consuming.
+* [Data transfer](https://rdmkit.elixir-europe.org/data_transfer.html) from the data providers to the data brokering platform will depend on the nature and volume of data. The volume might be larger if data providers can submit data in batches. 
```
```suggestion
* [Data transfer](data_transfer) from the data providers to the data brokering platform will depend on the nature and volume of data. The volume might be larger if data providers can submit data in batches. 
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Regular data checks should always be performed at the database level, to check unicity of identifiers where expected (e.g. sample identifier within a laboratory is expected to be unique) and identify potential incoherences in the data (e.g. division indicated in the name of a virus versus division indicated in the location field).
+* Data brokers should also consider implementing a mechanism for metadata and data files update within their platform, and define mechanisms to pass on the updates to international repositories. Please consider that depending on the update mechanisms in place at the international repositories (e.g. API vs manual update via email), this process might become quite time-consuming.
+* [Data transfer](https://rdmkit.elixir-europe.org/data_transfer.html) from the data providers to the data brokering platform will depend on the nature and volume of data. The volume might be larger if data providers can submit data in batches. 
+* Clarify with the data providers how the data will be processed, in terms of data curation/cleaning and downstream [data analyses](https://rdmkit.elixir-europe.org/data_analysis.html). 
```
```suggestion
* Clarify with the data providers how the data will be processed, in terms of data curation/cleaning and downstream [data analyses](data_analysis). 
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Data brokers should also consider implementing a mechanism for metadata and data files update within their platform, and define mechanisms to pass on the updates to international repositories. Please consider that depending on the update mechanisms in place at the international repositories (e.g. API vs manual update via email), this process might become quite time-consuming.
+* [Data transfer](https://rdmkit.elixir-europe.org/data_transfer.html) from the data providers to the data brokering platform will depend on the nature and volume of data. The volume might be larger if data providers can submit data in batches. 
+* Clarify with the data providers how the data will be processed, in terms of data curation/cleaning and downstream [data analyses](https://rdmkit.elixir-europe.org/data_analysis.html). 
+* [Data storage](https://rdmkit.elixir-europe.org/storage.html) needs should be carefully addressed; consider storing data in compressed formats and deleting intermediate files from analyses that could be recomputed if needed to gain storage space.
```
```suggestion
* [Data storage](storage) needs should be carefully addressed; consider storing data in compressed formats and deleting intermediate files from analyses that could be recomputed if needed to gain storage space.
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+
+## Sharing data to public repositories
+### Description
+Often, a goal of the broker is to submit the collected and harmonised data in an open archive for the benefit of both the data owners and the wider scientific community. This supports sharing data with collaborators, referencing for publication and provides a [long-term storage solution](https://rdmkit.elixir-europe.org/storage.html).
```
```suggestion
Often, a goal of the broker is to submit the collected and harmonised data in an open archive for the benefit of both the data owners and the wider scientific community. This supports sharing data with collaborators, referencing for publication and provides a [long-term storage solution](storage).
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+In the particular case of SARS-CoV-2 genomic data, more information will be available on the SARS-CoV-2 domain page (under construction, coming soon).
+
+### Considerations
+* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. See here for more information: https://rdmkit.elixir-europe.org/data_publication.html.
```
```suggestion
* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. More information can be found on our [data publication](data_publication) page.
```
urls should always be descriptive for accessibility reasons :)

Nazeeefa replied:
Got it! Thank you so much, @bedroesb!

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+### Considerations
+* There are many data repositories, so it is important to decide where you plan to share the data and adhere to the correct standards and submission protocols for your data type. This may notably include adhering to checklists provided by the chosen data repositories. See here for more information: https://rdmkit.elixir-europe.org/data_publication.html.
+* If submitting to open access repositories, ensure clarity on which fields and data to share and, if these fields contain sensitive information, should they be anonymised/pseudonymised
+  * metadata anonymisation:  [sensitive data](https://rdmkit.elixir-europe.org/sensitive_data.html), and [human data](https://rdmkit.elixir-europe.org/human_data.html)
```
```suggestion
  * metadata anonymisation:  [sensitive data](sensitive_data), and [human data](human_data)
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Submitted data may not be deleted, but may be suppressed from public view upon request
+* Submitted records may be updated at a later date with new or amended metadata entries. However, it should be noted that upon creation of a new record, data is tagged for distribution and selected metadata fields are exchanged with other repositories. Redistribution of updated records is usually not triggered automatically, so updating records fully can be a time consuming and manual process.
+* After data is submitted to a public repository, should the original copy of the data be retained at the central brokering platform and linked to its public counterpart? Or should it be removed and replaced with the ID of the public record?
+* Within INSDC databases, data may be submitted and held privately for a period of time by setting a release date at time of submission: https://ena-docs.readthedocs.io/en/latest/faq/release.html
```
```suggestion
* Within INSDC databases, data may be submitted and held privately for a period of time by setting a release date at time of submission, see [ENA documentation](https://ena-docs.readthedocs.io/en/latest/faq/release.html).
```

bedroesb commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Submitted records may be updated at a later date with new or amended metadata entries. However, it should be noted that upon creation of a new record, data is tagged for distribution and selected metadata fields are exchanged with other repositories. Redistribution of updated records is usually not triggered automatically, so updating records fully can be a time consuming and manual process.
+* After data is submitted to a public repository, should the original copy of the data be retained at the central brokering platform and linked to its public counterpart? Or should it be removed and replaced with the ID of the public record?
+* Within INSDC databases, data may be submitted and held privately for a period of time by setting a release date at time of submission: https://ena-docs.readthedocs.io/en/latest/faq/release.html
+* By default, data submitted using default accounts are accredited to the submitting institute. However, broker accounts have the ability to credit different institutes with different records: https://ena-docs.readthedocs.io/en/latest/faq/data_brokering.html 
+
+### Solutions
+* There are many tools available to remove human reads from your non-human data, e.g. https://github.com/alakob/Metagen-FastQC-Docker 
+* If submitting to ENA, please find the sample metadata checklist relating to your domain here: https://www.ebi.ac.uk/ena/browser/checklists 
+* For information on updating assemblies, see here: https://ena-docs.readthedocs.io/en/latest/update/assembly.html#updating-assemblies
+* For information on updating all other data types, see here: https://ena-docs.readthedocs.io/en/latest/submit/general-guide/programmatic.html#submission-xml-update-existing-objects
```
Same with these urls, try to avoid plain urls :)

bedroesb reviewed (changes requested):
updated some urls since it is advised to not use absolute linking towards internal pages (this way we are not sensitive to domain changes). I see you refer to multiple resources that are already in the tools table, could you at the data brokering tag to the [spreadsheet](https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668) of tools?

bedroesb commented:
Some information about [descriptive URLs](https://accessibility.oregonstate.edu/descriptivelinks#:~:text=The%20purpose%20of%20descriptive%20links,is%20vitally%20important%20and%20necessary)

floradanna commented:
@Nazeeefa Thanks for this contribution.

I would suggest to improve a bit the image:
- What are the empty squares with dark blue frame? Can this be made more clear in the image?
- Some icons have thicker then other. Could it be possible to you use icons more similar to each other?
- What is the rectangle with light blue frame? Does it represent also data providers? Could you make it more explicit please in the image.
- The light blue color is not very visible, maybe a color with more contrast?
- Could the arrows have the same color of the "origin" ? It would be easier to follow. See example below.
- Only Covid19 portal, GISAID and health authorities are present in the image. In my opinion, it gives the impression that this model only apply to health related domain. Maybe we could simply write "Local and international repositories" and then list few as examples. Also, Covid19 portal might not be the best example, since data are actually submitted to ENA, PRIDE, ArrayExpress etc... and only indexed by the portal.
- Is "authorities" in the caption the correct word? Maybe National health databases?
- The credits/licence of the icons could be written as separated paragraph under the caption or in smaller case/different font. I don't know. @bedroesb @martin-nc what do you think?

![Screenshot 2022-04-28 at 12 13 24](https://user-images.githubusercontent.com/51486716/165733153-0961cf1d-ed3b-41e2-a65c-258ce5ab79fc.png)

martin-nc commented:
1. **Graphic design:** I guess we should stick to the [style guide](https://rdmkit.elixir-europe.org/style_guide#graphic-design) and use the colours specified there, and the icons linked to from the Noun Project. We may need to choose another icon if we don't have one in the Noun Project collection. If there is a difference in style in the icon (line thickness, curved vs square corners) then we can fix that.
2. **Semantics:** I'm not too sure what it's showing, but maybe that's my ignorance! For my own understanding, I've stripped it down to a simpler form. Is it something like this:
![data-brokers](https://user-images.githubusercontent.com/15261745/165754425-4f95c0f0-0ec4-4476-823e-978fd7f2f74f.jpg)
The Google Drawing is here https://docs.google.com/drawings/d/1tCPrFs1jENmhQ9aS8sKkMQREz2jm6ljg39TmbHLt7EI/edit?usp=sharing. I'm not suggesting we use this, of course! It was just for me to think it through. I wasn't sure what the difference was between "Individual data producers" and "Data producer and data provider". Is it that there are two workflows, but that the data producer can be the same? If so, maybe they should both have the same label text?

alebrand commented:
Dear all, thanks a lot for all the comments on the image, I adapted the figure as suggested by @martin-nc , thank you very much.
I guess it's also fine to drop the icons, I took them all from Noun Project but they are from different authors hence the differences in thickness etc. Perhaps it's clearer now anyhow and the legend explains it? Regarding the wording, I'm fine having "public health databases". I actually meant sending data directly to public health authorities, but the term database is more generic and encompasses as well the fact of sending data to the local Ministry of public health who will store the data in some internal database.

Please find below a suggestion for the updated legend:

Figure 1: Data Brokering Workflow. Individual data producers may process the data, store it, and submit it directly to international repositories or public health databases. In the data brokering model instead, several data producers submit their data to a common data recipient that might be charge of curating the data, analysing it with common pipelines, storing it and re-sharing parts of the data (as agreed with the data providers) to public health databases and international repositories. The latter service is often referred to as “data brokering”, i.e. sharing data on behalf of others within a well defined ethical and legal framework. Note that legal aspects should be considered along all the steps.

Hoping it's fine like this, let me know. Best, Aitana

martin-nc commented:
@alebrand Thanks for updating the diagram, and sorry if it's been a hassle! If everyone is happy with it perhaps we can redraw it in a proper image editor so it looks nicer, and has the brand colours and fonts. I can do that if you like.

Can I suggest a light edit to the caption (just small grammar changes):

> Figure 1: Data Brokering Workflow. Individual data producers can process the data, store it, and submit it directly to international repositories or public health databases. Alternatively, in the data brokering model several data producers can submit their data to a common data recipient. This recipient might be charge of curating the data, analysing it with common pipelines, storing it, and re-sharing parts of the data to public health databases and international repositories  (as agreed with the data providers). The latter service is often referred to as “data brokering” i.e. sharing data on behalf of others within a well defined ethical and legal framework. Note that legal aspects should be considered along all the steps.

Nazeeefa commented:
Thank you @alebrand and @martin-nc 
Here are my few tiny suggestions to fix the text:

> **Figure 1: Data Brokering Workflow.** Alternatively, in the data brokering model,  several data producers can submit their data to a common data recipient. This recipient might be in charge of curating the data...

I suggest to bold "Figure 1" and figure title. I added comma after 'brokering model' so that sentence is easy to read, and finally I added "in" before charge of "curating the data..."

alebrand commented:
thank you @Nazeeefa  and @martin-nc all good with me with your caption suggestions, thanks!

Nazeeefa commented:
@martin-nc Kindly let me know once the latest version of figure is available and feel free to update/replace the caption based on our suggested changes, or let me know and I can do it. Thank you for your time.

martin-nc commented:
@Nazeeefa Sorry - I wasn't sure if I was supposed to do the diagram! What about this (not glamorous but it's at least in the brand colours and font!):

![data-brokerage](https://user-images.githubusercontent.com/15261745/168126148-d01d202c-10a6-4494-9b20-fcb5c7cf83e2.svg)

martin-nc commented:
Alternative version. I think we could add icons to this, too, if desired:


![Asset 6](https://user-images.githubusercontent.com/15261745/168181830-ec68670c-78a1-40d4-860f-3557b551cb99.svg)

Nazeeefa commented:
Thank you so much @martin-nc I am happy with both versions, wondering if @alebrand prefers second one and want to add icons to it? :)

Apparently, I am unable to figure out how to upload the figure to this PR. Kindly upload whichever one is preferred by @alebrand or let me know and I'll figure out.

alebrand commented:
> 

Dear all, thanks a lot, I think I prefer the first version with blank fills. Fine to go like this without icons, it makes the message more clear perhaps. Thank you very much for everything and for branding it!

floradanna commented:
@Nazeeefa I have few comments on the text, I don't know if you have seen them. It would be better if you could address them, please.

floradanna commented:
@Nazeeefa I have few comments on the text, I don't know if you have seen them. It would be better if you could address them, please.

alebrand commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+* Submitted records may be updated at a later date with new or amended metadata entries. However, it should be noted that upon creation of a new record, data is tagged for distribution and selected metadata fields are exchanged with other repositories. Redistribution of updated records is usually not triggered automatically, so updating records fully can be a time consuming and manual process.
+* After data is submitted to a public repository, should the original copy of the data be retained at the central brokering platform and linked to its public counterpart? Or should it be removed and replaced with the ID of the public record?
+* Within INSDC databases, data may be submitted and held privately for a period of time by setting a release date at time of submission, see [ENA documentation](https://ena-docs.readthedocs.io/en/latest/faq/release.html).
+* By default, data submitted using default accounts are accredited to the submitting institute. However, broker accounts have the ability to credit different institutes with different records: https://ena-docs.readthedocs.io/en/latest/faq/data_brokering.html 
```
Dear @floradanna 
Thanks for your comments. Requesting a broker account is indeed the starting point, and this implies different things and comes with different rights depending where you're submitting as a broker, e.g. to ENA or GISAID. We can update that section accordingly and perhaps merge part of this section with the Data publication page where this is relevant both for individual or broker submissions.

Regarding the SARS-CoV-2 domain page, it will cover many more aspects than just brokering, like metadata standards specific to SARS-CoV-2, legal aspects, technical infrastructures and dedicated tools in place... so definitely worth it, we see it as the page where we will document all the lessons learnt within ELIXIR-CONVERGE WP9.

Nazeeefa replied:
Thanks @floradanna for reviewing. We have added a section to the 'Data Publication' page https://github.com/elixir-europe/rdmkit/pull/942 Kindly have a look, I appreciate your time.

Nazeeefa commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,89 @@
+  * Need to remove any human data (e.g. reads) prior to submission
+* Submitted data may not be deleted, but may be suppressed from public view upon request
+* Submitted records may be updated at a later date with new or amended metadata entries. However, it should be noted that upon creation of a new record, data is tagged for distribution and selected metadata fields are exchanged with other repositories. Redistribution of updated records is usually not triggered automatically, so updating records fully can be a time consuming and manual process.
+* After data is submitted to a public repository, should the original copy of the data be retained at the central brokering platform and linked to its public counterpart? Or should it be removed and replaced with the ID of the public record?
```
Yes. We have added brief guidelines on brokering data submission and broker account.

floradanna commented:
@Nazeeefa thank you.
Please, make sure that:
- contributors are listed in the https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml (if they want to)
- this page is listed in related Data life cycle pages (e.g., sharing) and Role (e.g., IT, data manager)
- this page is listed in https://github.com/elixir-europe/rdmkit/blob/master/_data/news.yml (if you want)

alebrand commented on pages/your_tasks/data_brokering.md:
```diff
@@ -0,0 +1,74 @@
+title: Data brokering
+contributors: [Aitana Lebrand, Parul Tewatia, Wolmar Nyberg Åkerström, Carla Cummins, Nils Peder Willassen, Nazeefa Fatima]
+page_id: data brokering
+related_pages: 
```
short description looks nice to me, thanks.
For the contributors, @Nazeeefa , please use my new name, Aitana Neves, thank you.

floradanna commented:
@Nazeeefa you can merge! :)

aitananeves commented:
thanks a lot everyone!

bedroesb commented:
wait, is this done? :)

> -  contributors are listed in the https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml (if they want to)
> - this page is listed in related Data life cycle pages (e.g., sharing) and Role (e.g., IT, data manager)
> - this page is listed in https://github.com/elixir-europe/rdmkit/blob/master/_data/news.yml (if you want)

bedroesb commented:
we can also do it afterwards off  course

Nazeeefa commented:
> wait, is this done? :)
> 
> > * contributors are listed in the https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml (if they want to)
> > * this page is listed in related Data life cycle pages (e.g., sharing) and Role (e.g., IT, data manager)
> > * this page is listed in https://github.com/elixir-europe/rdmkit/blob/master/_data/news.yml (if you want)

@bedroesb the first two are done :) I'll do update news page once it is published :)

bedroesb commented:
I will add the brokering tools too now :)

Nazeeefa commented:
I noticed two of the links to ENA documentation were not working so I just updated with a few more changes (directly via GitHub) - hope that's fine?

bedroesb commented:
Sure, this is post merge ;)

Nazeeefa commented:
Hi @martin-nc, is it possible to make a tiny update to the figure? Apparently, the blue colour used for the figure (#3d68b0) is different from the one recommended by style guide (#376AC3). Thank you so much.

Nazeeefa commented:
Update to page authors: to stay consistent with template and writing style of other "Your Task" pages on RDMkit, we will change headings for this page to questions. The [template for "Your Task" pages](https://github.com/elixir-europe/rdmkit/blob/master/pages/your_tasks/TEMPLATE_your_tasks.md) currently says "Concrete problem 1, formulated as a question" for page section headings. 

What this means in practice: 
- the heading "Taking on the data broker role" can be changed to "How to take on the data brokering role?"
- "Collecting and processing the metadata and data" to "What are the best practices for collecting and processing data and metadata in data brokering?"
- "Sharing data to public repositories" to How can data brokers ensure effective sharing of data with public repositories?"

The suggested questions written above are just my suggestions, feel free to share what you would like to suggest :) Thanks, @wna-se @aitananeves @floradanna
